index.html

```diff
@@ -1209,6 +1262,13 @@ <h3>
             Updating the manifest
           </h3>
           <aside class="issue" data-number="446"></aside>
+          <p>
+            User agents MAY re-fetch and update a manifest for an installed web
+            application and apply updated manifest to its <a>application
+             context</a>. When a manifest is processed, user agents can
+            determine whether it's an installed application that can be updated
+            or a new application by comparing application's [=manifest/id=].
```

Comment:
I tried to explain how the id is used for manifest update. I am not sure if I should elaborate on current chrome manifest update lifecycle though as it's specific to chrome. WDTY? @dmurph @marcoscaceres

Reply:
Yeah I think leaving it intentionally vague is good. We don't want to impose any details on _how_ the update should work.

Reply:
I guess the only thing to add is maybe what to do if pre-processing and update fails. I guess nothing happens (i.e., just keep using the currently installed manifest) - but it would might be nice to just say that.

Reply:
@marcoscaceres should this be SHOULD behavior for the case you mentioned?

Reply:
yes, SHOULD would be good.